src/battle.ts

```diff
@@ -1187,10 +1196,12 @@ class Battle {
 	}
 	switchSides() {
 		this.setSidesSwitched(!this.sidesSwitched);
+		this.scene.updateSidebars();
```

Comment:
This is the wrong place for this. `resetToCurrentTurn` already redraws everything, including sidebars – if it doesn't, that's a bug and should be fixed, probably in `animationOn`.

Reply:
```suggestion
```